StateInfo.java

```diff
@@ -10,7 +10,7 @@ public StateInfo(String StateCapital, int StatePopulation, int StateSize, DLList
         this.StateCapital = StateCapital;
         this.StatePopulation = StatePopulation;
         this.StateSize = StateSize;
-        urls = new DLList<String>();
+        this.urls = urls;
```

Comment:
This I changed because it keeps throwing null pointer exceptions when I was testing out the `View Info` functionality. The reason the null pointer exceptions occur is entirely due to `urls = new DLList<String>();` and the way we call the constructor of `StateInfo` here:

https://github.com/zwlwilliamzha/HashMap-Lab/blob/166875adcafe4d71da0338dc4b660937adca7858/Screen.java#L80